lib/isorender/dom-shims.js

```diff
@@ -84,11 +84,14 @@ Document.prototype.createElement = function(tagName) {
   return el;
 };
 
-global.customElements = global.customElements || {
```

Comment:
No big deal but why remove capabilities? (in this case to define/shim `customElements` elsewhere before loading this module)

Reply:
Put it back, made the customElement registry not override itself. It seems mochapack is somehow initializing the module twice. So much state :(

Reply:
smh for mochapack